torch/csrc/jit/mobile/runtime_compatibility.cpp

```diff
@@ -8,5 +10,45 @@ uint64_t _get_runtime_bytecode_version() {
   return caffe2::serialize::kProducedBytecodeVersion;
 }
 
+/*
+ * Returns all registered PyTorch ops and their versioning
+ */
+std::unordered_map<std::string, OperatorInfo> _get_runtime_ops_and_info() {
+  std::unordered_map<std::string, OperatorInfo> result;
+
+  // Grab the jit operators
+  auto nonDispatcherOperators = torch::jit::getAllOperators();
+  for (const auto& full_op : nonDispatcherOperators) {
+    auto op = full_op->schema();
+    int num_schema_args = op.arguments().size();
+    if (op.overload_name() != "") {
+      result.emplace(
+          op.name() + "." + op.overload_name(), OperatorInfo{num_schema_args});
+    } else {
+      result.emplace(op.name(), OperatorInfo{num_schema_args});
+    }
+  }
+
+  // Grab the dispatcher operators
+  auto dispatcherOperators = c10::Dispatcher::singleton().getAllOpNames();
+  for (auto& op : dispatcherOperators) {
+    // grab schema
+    const auto op_handle = c10::Dispatcher::singleton().findOp(op);
+    int num_schema_args = NO_SCHEMA;
+    if (op_handle->hasSchema()) {
```

Comment:
@raziel @iseeyuan What do you think about not returning ops with no schema from this api and instead just collecting them and then throwing a torch warn that lists all the ops with no schema?

Reply:
I think it may make sense to let caller know what's exactly inside the runtime. Returning all ops, with/wo schema makes more sense to me.

Reply:
I don't understand well the no-schema situation.

* Are these no-schema ops valid ops but that lack the metadata? i.e. can they be used to run stuff even without the schema? If so then it seems we should include them in the set.
* I think you're investigating this, but why do we have these no-schema ops? Is this something we think we can/will fix soon?

Reply:
@raziel For your first question, yes the no-schema ops are "valid" ops that can be used. The kernel is there but the schema is stripped. For your second question, I'll do some experiment and see if there's a fix.

Reply:
yeah so since no-schema ops are still valid ops then we should return them in the API.

Separately we should figure out why they have no schema.